services/test-data/dataset_defs/ndvi (3587).json

```diff
@@ -1,51 +1,53 @@
 {
   "properties": {
     "id": {
-      "internal": "a3f6955e-53cc-4752-9e4a-e9f0493add80"
+      "type": "internal",
+      "dataset": "a3f6955e-53cc-4752-9e4a-e9f0493add80"
     },
     "name": "NDVI3857",
     "description": "NDVI data from MODIS reprojected to EPSG:3857",
     "sourceOperator": "GdalSource"
   },
   "metaData": {
-    "GdalMetaDataRegular": {
-      "resultDescriptor": {
-        "dataType": "U8",
-        "spatialReference": "EPSG:3857",
-        "measurement": "unitless",
-        "noDataValue": 0.0
+    "type": "gdalMetaDataRegular",
```

Comment:
looks weird, but I guess it's okay(?)

Reply:
it's at least consistent with all the other fields that are camelCase. We could make an exception for operators (which are still Capitalized I believe) and meta data. But where do we draw the line then?